### ISSUE TITLE
Handle empty lists when @BindIn

### ISSUE THREAD

**Issue by mbarbieri** [jdbi/jdbi#160]
When binding a list using `@BindIn`, if you pass an empty collection the generated sql will be like:
```sql
select * from table where something in ()
```
that (in PostgreSQL) gives you an exception
```
ERROR:  syntax error at or near ")"
```
At least in PostgreSQL generating this will work as expected
```sql
select * from table where something in (null)
```

**Comment by Neilpoulin** [comment]
I have also run into this same issue

**Comment by stevenschlansker** [comment]
Fixed by #473

**Comment by maximdim** [comment]
Is this fix included in 2.77? I'm running against Postgresql:

```
@SqlQuery("select * from student where id in (<ids>)")
abstract List<Student> findByIds(@BindIn(value="ids", onEmpty=EmptyHandling.NULL) Collection<String> ids);
```

and getting the exception:

```
org.skife.jdbi.v2.exceptions.UnableToExecuteStatementException: org.postgresql.util.PSQLException: ERROR: syntax error at or near ")"
  Position: 46 [statement:"select * from student where id in (<ids>)", located:"select * from student where id in ()", rewritten:"select * from student where id in ()", arguments:{ positional:{}, named:{__ids_0:'null'}, finder:[]}]
	at org.skife.jdbi.v2.SQLStatement.internalExecute(SQLStatement.java:1338)
```

**Comment by stevenschlansker** [comment]
@maximdim You might be hitting https://github.com/jdbi/jdbi/pull/558 ?  We'll have that fixed shortly, sorry.
